sample-bugsnag/shared/src/iosMain/kotlin/co/touchlab/kermitsample/CrashIntegration.kt

```diff
@@ -8,12 +8,11 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-include(/*":app", ":app-browser",*/ ":shared")
+package co.touchlab.kermitsample
```

Comment:
package above for android app is `co.touchlab.kermitsamplecrashlog`. Should we fix something here or there?